src/main/resources/page/js/dummy-content-displayer.js

```diff
@@ -0,0 +1,7 @@
+class WriteOutDataOnDummyPage {  
+    constructor() {}
+
+    static writeOutDateOrTime(id, data) {
```

Comment:
We can change the name of the method similarly just as we can do in the case of the class name.